content/F2015/F15-event-career-panel.md

```diff
@@ -0,0 +1,21 @@
+title: CSC and WiCS Career Panel
+Date: 2015-09-24 17:00-19:00
```

Comment:
I get the idea of what you're saying (keeping the Orientation post as featured) but can you explain a bit further what posting date is and how the featured post is chosen based on it?